benchmarks/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@ find_package(benchmark 1.5.4 REQUIRED)
 message(STATUS "Found benchmark: ${benchmark_DIR} (version \"${benchmark_VERSION}\")")
 
 add_subdirectory(brute_force_vs_bvh)
-add_subdirectory(bvh_driver)
+if(NOT WIN32)
```

Comment:
Please add a `FIXME` with a suitable comment describing we this gets disabled here.